rclcpp/include/rclcpp/service.hpp

```diff
@@ -486,6 +486,14 @@ class Service
   {
     rcl_ret_t ret = rcl_send_response(get_service_handle().get(), &req_id, &response);
 
+    if (ret == RCL_RET_TIMEOUT) {
+      RCLCPP_WARN(
+        rclcpp::get_node_logger(node_handle_.get()).get_child("rclcpp"),
+        "failed to send response: %s",
```

Comment:
just a suggestion, it would be clear for user that timeout happened?

```suggestion
        "failed to send response because of timeout: %s",
```

Reply:
Yes, agreed that it would be clearer.